CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 * Wrong assertion for user setting's `width` property when creating a new detail in separate route.
 * Rolling back isDeleted state of model if errors occurrs during destroying of the model.
 
+### Removed
+* Removed outdated style in `object-list-view` styles component.
```

Comment:
I think this should be no past time. Like this:
"### Fixed
- Fix typos in recent README changes.".
M?